### PR TITLE
fix: add italian translations

### PR DESCRIPTION
**Pull request by czosel**

### CODE REVIEW

**Comment by luytena** [comment]
Were the terms translated or taken from the official [Catalogo delle caratteristiche (Merkmalskatalog)](https://www.bfs.admin.ch/bfs/de/home/statistiken/kataloge-datenbanken.assetdetail.22905272.html)?

File: translations/it.yaml [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+      heading: Collega via
+      search: Cerca via...
+      linkSuccess: Via collegata con successo.
+      linkError: Errore nel tentativo di collegare la via.
+      streetHint: Per registrare un'entrata occorre collegare una via.
+      searchHint: 'Nella casella di ricerca è possibile inserire un asterisco (*) come segnaposto per cercare parti del nome della via.'
+      searchHintExamples: Esempi
+      searchHintExamples1: 'Unter* : Unterfeldweg, Unterschlossweg'
+      searchHintExamples2: '*weg : Waldweg, Unterschlossweg'
+      searchHintExamples3: 'B*strasse : Bernstrasse, Bachstrasse'
```
Comment (luytena) [inline translations/it.yaml:338]: I think here we used locale specific examples

File: translations/it.yaml [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+    loginModal:
+      header: Effettua login nel REA
+      login: Login
+      password: Password...
+      username: Nome utente...
+      municipality: Seleziona un comune...
+      municipalitySearch: Cerca comune...
+      municipalityNoneFound: Non è stato trovato alcun comune
+      loginInfo: "<b>{user}</b> login effettuato per <b>{municipality}</b>"
+      loginIssueContact: "<b>Se non è possibile effettuare il login contattare direttamente...</b><br><br>hotline<br>housing-stat@bfs.admin.ch<br>0800 866 600<br>Dal lunedì al venerdì<br>08.30-11.30 e 14.00-16.00"
```
Comment (luytena) [inline translations/it.yaml:239]: Does the BfS have language specific telephone numbers?

**Comment by luytena** [comment]
Looks good to me, but I just briefly scrolled through it :)

File: translations/it.yaml (hunk 2 above)
Comment (czosel) [reply translations/it.yaml:239]: Seems to be the same number in italian: https://www.housing-stat.ch/it/index.html

File: translations/it.yaml (hunk 1 above)
Comment (czosel) [reply translations/it.yaml:338]: That makes sense. I'd prefer to fix this in a future MR, so that the customer can make some fitting suggestion. Ok?

**Pull request merged**